config.go

```diff
@@ -614,9 +622,16 @@ func DictConfig(conf *Conf) error {
 			if err != nil {
 				return err
 			}
-			handler, err = NewSyslogHandler(priority, tag)
-			if err != nil {
-				return err
+			if network != "" && raddr != "" {
+				handler, err = NewSyslogHandlerToAddr(network, raddr, priority, tag)
+				if err != nil {
+					return err
+				}
+			} else {
+				handler, err = NewSyslogHandler(priority, tag)
+				if err != nil {
+					return err
+				}
```

Comment:
Same code is line 627~629. Please move it to the right bracket after line 635, to remove the duplication.

Reply:
@hhkbp2 , I have addressed the feeback.